contributor/03-contributor-ethos-script.md

```diff
@@ -10,3 +10,87 @@ A Contributor to an InnerSource project is like a guest in a home.  How should a
 
 ## Script
 
+X: In the previous video section we took a closer look at when to become an
+InnerSource contributor. We also investigated which types of contributions
+InnerSource projects typically are looking for. In this video you will learn
+more about how to be a good contributor following the guest-in-home analogy.
+
+X: So you are planning to support an InnerSource project of your choice. How do
+you get started? InnerSource contributors understand that essentially they are
+guests in the house of the host team. When visiting your neighbors - even if
+their door is open, you probably won't enter without knocking on the door first.
+
+The same is true for a lot of InnerSource projects: Inviting contributions from
+the outside does not imply giving write access to everyone right from the start.
+Instead you will submit your changes through a pull request and have them
```

Comment:
I wonder if it would make any sense to hint at non-PR style models such as good ol' Gerrit or if people are expected to bring that piece of transfer themselves. (I'd hope so, just bringing it up for completeness.)  

```suggestion
Instead you will submit your changes through a pull request or proposed patchset and have them
```